### DISCUSSION TITLE
[Feature] Journal Entries

### DISCUSSION THREAD

**Discussion by budowski** [immich-app/immich#14300] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Adding dated journal entries to Immich - matching between photos/videos of specific dates, with textual journal entries.
Extra features for this:
* Having rich text formatting for these textual entries
* Importing journal entries from other sources (DB/etc)
* Showing these entries as part of the memories flow

### Platform

- [x] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
I'm closing this as a dupe of #3896. Immich is not a journaling app.

**Reply by mr-kristoffer** [reply]
Was thinking, this would be a cool feature as well, since I am planning making immich our platform for storing family memories.
